### DISCUSSION TITLE
How to get a Thumbnails Frontend Path in Console Command?

### DISCUSSION THREAD

**Discussion by dasraab** [pimcore/pimcore#12326] [Q&A / Support]
I'm using Pimcore X with S3 Asset Storage and "frontend_prefixes" - everything is working fine so far.

I'm pushing new BlogEntries (DataObject) automatically into a Meilisearch-Server. Each BlogEntry has a teaserImage.

I want to push the frontend Path of the teaserImage's Thumbnail to Meilisearch. This should happen inside a Console Command.

I'm using 
> $object->getTeaserImage()->getThumbnail("blog-listing')->getPath(false);

to get the Thumbnail

The Problem:
It seems like there is no way of getting the frontend Path of a Thumbnail inside a Console Command as Tool::isFrontend() always returns false in Console-Mode.

See 
[This is where the Path is generated](https://github.com/pimcore/pimcore/blob/90a603647ae074be3b99b545b8246a4a59244fdd/models/Asset/Thumbnail/ImageThumbnailTrait.php#L303)
[This is where Pimcore checks if its a frontend Request](https://github.com/pimcore/pimcore/blob/90a603647ae074be3b99b545b8246a4a59244fdd/lib/Tool.php#L297)

Question:
Is there a way to force the getThumbnail() or getPath() methods to return the frontend path of a Thumbnail?

**Comment by dasraab** [comment]
any updates on this?

**Comment by dvesh3** [comment]
It will most probably be covered in https://github.com/pimcore/pimcore/issues/11692
